### PR TITLE
Add audio settings from OBS

### PR DESCRIPTION
**Pull request by eddyStreamlabs**

### CODE REVIEW

**Comment by eddyStreamlabs** [comment]
Requires an update of osn.

File: app/services/settings/settings.ts [hunk 1]
```diff
@@ -303,7 +304,10 @@ export class SettingsService extends StatefulService<ISettingsState>
   }
 
   setSettings(categoryName: string, settingsData: ISettingsSubCategory[]) {
-    if (categoryName === 'Audio') return this.setAudioSettings(settingsData);
+    if (categoryName === 'Audio') {
+      this.setAudioSettings([settingsData[1]]);
+      settingsData.pop();
```
Comment (gettinToasty) [inline app/services/settings/settings.ts]: why are you mutating the settingsData here? if settingsData[1] is the last element you could write
```
this.setAudioSettings([settingsData.pop()])
```

but this seems kind of weird regardless, why not just pass in the single element as the parameter here?

Comment (eddyStreamlabs) [reply app/services/settings/settings.ts]: The second element of the array needs to be passed to `setAudioSettings`, then the rest of the `setSettings` function should only do treatment with the first element of the array.

Buy you"re right, I can do what you suggest.

Comment (eddyStreamlabs) [reply app/services/settings/settings.ts]: Updated

**Pull request merged**